.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -0,0 +1,74 @@
+jobs:
+- job: osx_64
+  pool:
+    vmImage: macOS-10.13
+  strategy:
+    maxParallel: 8
+    timeoutInMinutes: 240
+  steps:
+  # TODO: Fast finish on azure pipelines?
+  - script: |
+      echo "Fast Finish"
+      
+
+  - script: |
+      echo "Removing homebrew from Azure to avoid conflicts."
+      curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew
+      chmod +x ~/uninstall_homebrew
+      ~/uninstall_homebrew -fq
+      rm ~/uninstall_homebrew
+    displayName: Remove homebrew
+
+  - script: |
+      echo "Installing Miniconda"
+      set -x -e
+      curl -o $(Build.StagingDirectory)/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh
+      chmod +x $(Build.StagingDirectory)/miniconda.sh
+      $(Build.StagingDirectory)/miniconda.sh -b -p $(Build.StagingDirectory)/miniconda
+      export PATH=$(Build.StagingDirectory)/miniconda/bin:$PATH
+      echo "Setting up Conda environment"
+    displayName: 'Install miniconda'
+
+  - script: |
+      export PATH=$(Build.StagingDirectory)/miniconda/bin:$PATH
+      set -x -e
+      conda install --yes --quiet conda-forge::conda-forge-ci-setup=2
+    displayName: 'Add conda-forge-ci-setup=2'
+
+  - script: |
+      set -x -e
+      export PATH=$(Build.StagingDirectory)/miniconda/bin:$PATH
+      echo "Configuring conda."
+
+      setup_conda_rc ./ ./recipe ./.ci_support/${CONFIG}.yaml
+
+      source run_conda_forge_build_setup
+      conda update --yes --quiet --override-channels -c conda-forge -c defaults --all
+    env: {
+      OSX_FORCE_SDK_DOWNLOAD: "1"
+    }
+    displayName: Configure conda and conda-build
+
+  - script: |
+      # Run the travis script to build
+      source ./.travis_scripts/create_feedstocks;
```

Comment:
This script creates feedstocks

Reply:
Fixed

Reply:
Do we want to explore migrating feedstock creation to Azure?